lib/responders.rb

```diff
@@ -1,11 +1,5 @@
-require 'action_controller'
```

Comment:
Good catch!

Do we really need the changes in this file? I think the one in `controller_method.rb` is actually the issue. :)